app/middlewares/user.js

```diff
@@ -0,0 +1,16 @@
+const { check } = require('express-validator/check');
+
+export function createValidation() {
+  return [
+    check('firstName').exists(),
+    check('lastName').exists(),
+    check('password')
+      .exists()
+      .isLength({ min: 8 })
+      .isAlphanumeric(),
+    check('email')
+      .exists()
+      .isEmail()
+      .matches(new RegExp(`@wolox(${['\\.com\\.ar', 'co', '\\.cl', '\\.mx'].join('|')})$`))
```

Comment:
This is ok for now, but later we would change this for schemas in middlewares card 👀

Reply:
But this Regex is amazing and you know it

Reply:
🤔